src/components/ChatEntry.js

```diff
@@ -1,22 +1,59 @@
 import React from 'react';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
 import PropTypes from 'prop-types';
 
-const ChatEntry = (props) => {
+const ChatEntry = ({
+  sender,
+  body,
+  timeStamp,
+  id,
+  liked,
+  localName,
+  localColor,
+  remoteColor,
+  onUpdate,
+}) => {
+  const toggleLiked = () => {
+    const updatedEntry = {
+      sender,
+      body,
+      timeStamp,
+      id,
+      liked: !liked,
+    };
+    onUpdate(updatedEntry);
+  };
+
+  const senderClass = sender === localName ? 'local' : 'remote';
+  const colorClass = sender === localName ? localColor : remoteColor;
```

Comment:
👍  Nice way of handling the likes and colors logic using a helper function and ternary operators!